C++/ds/Priority_Queue.cpp

```diff
@@ -0,0 +1,129 @@
+#include<iostream>
+#include<unordered_map>
+#include<vector>
+#include<algorithm>
+
+using namespace std;
+
+// MAX HEAP
+class pQueue {
+    private:
+        int heapSize;
+        unordered_map <int, vector<int>> heapMap;
+        vector <int> heapElements;
+
+    public:
+        pQueue() {
+            heapSize = 0;
+        }
+
+        void add(int element) {
+            heapSize++;
+            heapElements.push_back(element);
+            if(heapMap.find(element)==heapMap.end()){
+                heapMap[element] = vector <int> ({heapSize-1});
+            } else {
+                heapMap[element].push_back(heapSize-1);
+            }
+            maintainLoopVariant(heapSize-1);
```

Comment:
Space should be added before and after each operator and operand. Please make this change.